Src/DependencyCollector/NuGet/DependencyCollector.NuGet.csproj

```diff
@@ -16,7 +16,7 @@
     <NoWarn>2008</NoWarn>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <PackageSpecFile>$(MSBuildProjectDirectory)\Package.nuspec</PackageSpecFile>
-    <PackageVersionFile>$(BinRoot)\$(Configuration)\Src\Web\Web.Net45\Microsoft.AI.Web.dll</PackageVersionFile>
+    <PackageVersionFile>$(BinRoot)\$(Configuration)\Src\Web\Web\net45\Microsoft.AI.Web.dll</PackageVersionFile>
```

Comment:
no need in this property after the switch. It is used by old way NuGets were packaged